torch/_inductor/ir.py

```diff
@@ -4024,11 +4022,12 @@ class ChoiceCaller:
     Children classes: TritonTemplateCaller, CUDATemplateCaller.
     """
 
-    def __init__(self, name, input_nodes, layout):
+    def __init__(self, name, input_nodes, layout, description):
         super().__init__()
         self.name = name
         self.layout = layout
         self.input_nodes = input_nodes
+        self.description = description
```

Comment:
Do you mind documenting it? A type would also be good

Reply:
yeah it's just a string